pkg/reader/ccip.go

```diff
@@ -625,6 +625,17 @@ func (r *ccipChainReader) GetRMNRemoteConfig(
 	}, nil
 }
 
+func (r *ccipChainReader) isValidAddress(address []byte, contractName string, chain cciptypes.ChainSelector) bool {
+	if len(address) == 0 {
+		r.lggr.Errorw("address for contract is empty",
```

Comment:
Don't use error, it will spam the logs with a big stack trace. Warn here makes more sense to me

Reply:
Hmm, I would say it depends. If this is something that should never happen or happens because of the wrong configuration then error/critical is easier to spot in the logs. If this is a regular check that can fail from time to time then definitely should not be `error`

Reply:
I agree with @mateusz-sekara here. If this error was present I could've figured out somethings earlier.